src/Sepes.Functions/host.json

```diff
@@ -2,9 +2,9 @@
     "version": "2.0",
   "logging": {
     "logLevel": {
-      "default": "Debug",
-      "System": "Debug",
-      "Microsoft": "Debug"
+      "default": "Information",
+      "System": "Warning",
+      "Microsoft": "Warning"
     },
     "applicationInsights": { 
```

Comment:
What does this do?

Reply:
Determines how detailed the logging from the function app is. We set it to debug to investigate something the other day, but it was just to much